pkgs/development/python-modules/awscrt/default.nix

```diff
@@ -33,6 +33,11 @@ buildPythonPackage rec {
     "strictoverflow"
   ];
 
+  postPatch = ''
+    substituteInPlace setup.py \
+      --replace "extra_link_args += ['-Wl,-fatal_warnings']" ""
```

Comment:
Did you try dropping Werror instead? https://github.com/awslabs/aws-crt-python/blob/v0.13.11/setup.py#L300

Reply:
Werror is complier flag while the error is thrown by ld, I've tried to build without Werror and it faled
```
ld: fatal warning(s) induced error (-fatal_warnings)
clang-11: error: linker command failed with exit code 1 (use -v to see invocation)
error: command '/nix/store/630b4hj9z3gj62c4fw3q7sxy1akql5l4-clang-wrapper-11.1.0/bin/clang' failed with exit code 1
```